### PR TITLE
Eliminate some TODOs for 0.14

### PR DESCRIPTION
**Pull request by sciunto**
I killed a couple of TODO items in the list. Please review, I leave the rest to someone else :)

### CODE REVIEW

**Comment by soupault** [comment]
This kind of PRs are much easier to review when they are organized in 1_commit/1_case way. Just saying :)

**Comment by sciunto** [comment]
I realized that as well when I submitted the PR...

File: skimage/restoration/_denoise.py [hunk 1]
```diff
@@ -111,14 +110,6 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
                              "``multichannel=True`` for 2-D RGB "
                              "images.".format(image.shape))
 
-    if sigma_range is not None:
```
Comment (soupault) [inline skimage/restoration/_denoise.py]: I've just discovered that it is still named as `sigma_range` in .pyx-file ...

**Comment by sciunto** [comment]
hold on a second, I discovered something.

**Comment by sciunto** [comment]
I discovered that a task about `compare_ssim` was missing in the TODO list :(

File: skimage/restoration/_denoise.py (hunk 1 above)
Comment (sciunto) [inline skimage/restoration/_denoise.py]: yep! I'm fixing it.

**Comment by sciunto** [comment]
@scikit-image/core Anyone for a second pair of eyes please?

**Comment by jni** [comment]
Very nice, thanks @sciunto! =)

**Comment by sciunto** [comment]
many thanks for reviewing @soupault and @jni

**Pull request merged**